enhancements/installer/ipi-publishing-status.md

```diff
@@ -0,0 +1,198 @@
+---
+title: expose-cluster-publishing-status
+authors:
+  - "@flavianmissi"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@patrickdillon, installer"
+  - "@Miciah, edge networking"
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - TBD
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "@bparees"
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - [IR-302](https://issues.redhat.com/browse/IR-302)
+---
+
+# Expose Cluster Publishing Status
+
+## Summary
+
+IPI clusters can be configured to be public or private. This is done in the
+install-config.yaml using the `Publish` parameter. There is no current standard
+defined for how components should access this parameter to bootstrap themselves
+as public (External) or private (Internal).
+
+## Motivation
+
+A component should be able to know how to bootstrap itself according to user
+preference. A central place providing the cluster publishing status at install
+time makes more sense than the installer having to individually configure
+various components.
+
+### User Stories
+
+* As an operator author, I want to easily discover the cluster publishing
```

Comment:
can we illustrate with a concrete example here?

Reply:
Thanks for reviewing @enxebre - I gave my own use case as example, let me know if it works.